docs/development/setup-dapr-development-env.md

```diff
@@ -1,57 +1,91 @@
+
+3. Install [golangci-lint](https://golangci-lint.run/usage/install).
 
 ### Windows
 
 1. The Go language environment [(instructions)](https://golang.org/doc/install#windows).
    Make sure that your GOPATH and PATH are configured correctly - You may set environment variables through the "Environment Variables" button on the "Advanced" tab of the "System" control panel. Some versions of Windows provide this control panel through the "Advanced System Settings" option inside the "System" control panel.
-   ```
+
+   ```powershell
    GOPATH=%USERPROFILE%\go
    PATH=%GOPATH%\bin;...
    ```
+
 2. [Delve](https://github.com/go-delve/delve/tree/master/Documentation/installation) for Go debugging
-    Delve for Go (NOTE: this is NOT Delve from the Office365) is a debugger utility is a debugger for the Go programming language.  Please refer to [Delve Home Page](https://github.com/go-delve/delve) for a detailed explanation.
+   Delve for Go (NOTE: this is NOT Delve from the Office365) is a debugger utility is a debugger for the Go programming language.  Please refer to [Delve Home Page](https://github.com/go-delve/delve) for a detailed explanation.
+
 3. [Git for Windows](https://gitforwindows.org)
-   * Install [Git with chocolatey](https://chocolatey.org/packages/git) and ensure that Git bin directory is in PATH environment variable
+   - Install [Git with chocolatey](https://chocolatey.org/packages/git) and ensure that Git bin directory is in PATH environment variable
+
     ```powershell
     choco install git -y --package-parameters="/GitAndUnixToolsOnPath /WindowsTerminal /NoShellIntegration"
     ```
-4. [MinGW](http://www.mingw.org/)
-  Install [MinGW v8.1.0 with chocolatey](https://chocolatey.org/packages/mingw/8.1.0) and ensure that MinGW bin directory is in PATH environment variable. Version 8.1.0 is the last version that installs mingw32-make.exe.
+
+4. [MinGW](https://sourceforge.net/projects/mingw/files/MinGW)
+   Install [MinGW v8.1.0 with chocolatey](https://chocolatey.org/packages/mingw/8.1.0) and ensure that MinGW bin directory is in PATH environment variable. Version 8.1.0 is the last version that installs mingw32-make.exe.
 
     ```powesrshell
     choco install mingw --version 8.1.0
     ```
+
 5. Rename mingw32-make.exe to make.exe from an administrative PowerShell.
 
     ```powershell
     copy C:\ProgramData\chocolatey\lib\mingw\tools\install\mingw64\bin\mingw32-make.exe C:\ProgramData\chocolatey\lib\mingw\tools\install\mingw64\bin\make.exe
     ```
 
-## Setup Kubernetes development environment
+## Setup a Kubernetes development environment
+
+Follow the guide on [how to set up a Kubernetes cluster for Dapr](https://docs.dapr.io/operations/hosting/kubernetes/cluster/).
+
+For development purposes, you will also want to follow the optional steps to install [Helm 3.x](https://helm.sh/docs/intro/install/).
+
+### Installing Make
```

Comment:
For no-k8s env, it still needs.

Reply:
Thanks for the catch, this was incorrectly nested under the previous header!